### ISSUE TITLE
Namespace not specified error

### ISSUE THREAD

**Issue by MG-LSJ** [mannprerak2/nearby_connections#72]
I get this error when building

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':nearby_connections'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 1s
```

**Comment by mannprerak2** [comment]
How exactly are you building?

Can you try running the example app on your device?

**Comment by MG-LSJ** [comment]
> How exactly are you building?
> 
> Can you try running the example app on your device?

I am using gradel version 8.2

**Comment by MG-LSJ** [comment]
> Will need to look into this, if you have a fix please file a PR.

Updated gradle to 8.0 (flutter 3.19 default) and implemented namespace in my [fork](https://github.com/MG-LSJ/nearby_connections). 

Might open a PR if you wish, only need to fix some deprecated warnings:

```
C:\Users\laksh\Documents\programming\nearby_connections\android\src\main\java\com\pkmnapps\nearby_connections\NearbyConnectionsPlugin.java:44: warning: [deprecation] Registrar in PluginRegistry has been deprecated
import io.flutter.plugin.common.PluginRegistry.Registrar;
                                              ^
C:\Users\laksh\Documents\programming\nearby_connections\android\src\main\java\com\pkmnapps\nearby_connections\NearbyConnectionsPlugin.java:53: warning: [deprecation] Registrar in PluginRegistry has been deprecated
	private static PluginRegistry.Registrar pluginRegistrar;
	                             ^
C:\Users\laksh\Documents\programming\nearby_connections\android\src\main\java\com\pkmnapps\nearby_connections\NearbyConnectionsPlugin.java:66: warning: [deprecation] Registrar in PluginRegistry has been deprecated
	public static void registerWith(Registrar registrar) {
	                                ^
C:\Users\laksh\Documents\programming\nearby_connections\android\src\main\java\com\pkmnapps\nearby_connections\NearbyConnectionsPlugin.java:285: warning: [deprecation] getAuthenticationToken() in ConnectionInfo has been deprecated
			args.put("authenticationToken", connectionInfo.getAuthenticationToken());
			                                              ^
C:\Users\laksh\Documents\programming\nearby_connections\android\src\main\java\com\pkmnapps\nearby_connections\NearbyConnectionsPlugin.java:332: warning: [deprecation] getAuthenticationToken() in ConnectionInfo has been deprecated
			args.put("authenticationToken", connectionInfo.getAuthenticationToken());
			                                              ^
C:\Users\laksh\Documents\programming\nearby_connections\android\src\main\java\com\pkmnapps\nearby_connections\NearbyConnectionsPlugin.java:389: warning: [deprecation] asJavaFile() in com.google.android.gms.nearby.connection.Payload.File has been deprecated
					args.put("filePath", payload.asFile().asJavaFile().getAbsolutePath());
					                                     ^
6 warnings
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```

**Comment by mannprerak2** [comment]
> Might open a PR if you wish

Sure, please go ahead. You can skip the depreciation warnings if they are a bit complicated to resolve. I will take a look as soon as I get some time.

**Comment by mannprerak2** [comment]
Closed via #73
